src/main_app/db/db_Templates.py

```
@@ -135,49 +152,56 @@ def _fetch_by_title(self, title: str) -> TemplateRecord:
+
+        title = _data["title"]
 
         try:
             # Use execute_query to allow exception to propagate
             self.db.execute_query(
                 f"""
-                INSERT INTO templates ({', '.join(add_fields)})
-                VALUES ({', '.join(["%s" for x in add_fields])})
+                INSERT INTO templates ({", ".join(add_fields)})
+                VALUES ({", ".join(["%s" for x in add_fields])})
                 """,
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)